projects/observability/src/shared/dashboard/data/graphql/api-error-percentage/api-error-percentage-data-source.model.ts

```diff
@@ -30,6 +30,12 @@ export class ApiErrorPercentageDataSourceModel extends GraphQlDataSourceModel<Me
     );
   }
 
+  @ModelProperty({
```

Comment:
Is `required: true` by default in ModelProperty?

Reply:
nope. added it.